presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueryFramework.java

```diff
@@ -306,7 +322,11 @@ protected void assertAccessDenied(
                 fail("Expected " + AccessDeniedException.class.getSimpleName());
             }
             catch (RuntimeException e) {
-                assertExceptionMessage(sql, e, ".*Access Denied: " + exceptionsMessageRegExp);
+                String query = format("Query: %s", sql);
```

Comment:
No need to use string format for simple concatenation
```java
String query = "Query: " + sql;

Reply:
done